### PR TITLE
Add TFC example

### PR DESCRIPTION
**Pull request by aliscott**

### CODE REVIEW

File: examples/terraform-cloud/README.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+      - name: Setup Infracost
+        uses: infracost/actions/setup@v1
+        with:
+          api-key: ${{ secrets.INFRACOST_API_KEY }}
+          
+      - name: Run Infracost
+        run: infracost breakdown --path=examples/terraform-cloud/code --format=json --out-file=/tmp/infracost.json
+        env:
+          INFRACOST_TERRAFORM_CLOUD_TOKEN: ${{ secrets.TFC_TOKEN }} # TODO: can be removed once https://github.com/infracost/infracost/pull/1148 is released
```
Comment (alikhajeh1) [inline examples/terraform-cloud/README.md:30]:
```suggestion
          # TODO: the following two envs be removed once https://github.com/infracost/infracost/pull/1148 is released in v0.9.14 of the CLI (https://github.com/infracost/infracost/releases)
          INFRACOST_TERRAFORM_CLOUD_TOKEN: ${{ secrets.TFC_TOKEN }} 
          # INFRACOST_TERRAFORM_CLOUD_HOST: https://my_tfe_host.com # For Terraform Enterprise users only.
```

File: testdata/terraform-cloud_comment_golden.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+      Monthly cost depends on usage
+        +$0.0000166667 per GB-seconds
+
+Monthly cost change for infracost/actions/examples/terraform-cloud/code
+Amount:  +$743 ($0.00 → $743)
+
+──────────────────────────────────
+Key: ~ changed, + added, - removed
+
+2 cloud resources were detected, rerun with --show-skipped to see details:
```
Comment (alikhajeh1) [inline testdata/terraform-cloud_comment_golden.md:62]: I wasn't expecting to see `--show-skipped` as we output that in the comments task. I probably messed-up somewhere and didn't add that to the test automation?

File: examples/terraform-cloud/README.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      
+      - name: Install terraform
+        uses: hashicorp/setup-terraform@v1
+        with:
+          terraform_wrapper: false # This is required so the `terraform show` command outputs valid JSON
+          cli_config_credentials_token: $${{ secrets.TFC_TOKEN }}
```
Comment (alikhajeh1) [inline examples/terraform-cloud/README.md:20]:
```suggestion
          cli_config_credentials_token: $${{ secrets.TFC_TOKEN }}
          # cli_config_credentials_hostname: https://my_tfe_host.com # For Terraform Enterprise users only
```

File: examples/terraform-cloud/README.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
+# Terraform cloud
```
Comment (alikhajeh1) [inline examples/terraform-cloud/README.md:1]:
```suggestion
# Terraform Cloud/Enterprise

This example shows how to run Infracost actions with Terraform Cloud and Terraform Enterprise. It assumes you have set a GitHub repo secret for the Terraform Cloud token (`TFC_TOKEN`). This token is used by the Infracost CLI to fetch the plan JSON from Terraform Cloud and use it to generate the cost estimate comment.

In the future, we'll add an example of how you can trigger the Infracost actions from Terraform Cloud's GitHub status checks.
```

Can you please edit the description to clarify which plan is fetched to generate the cost estimate? Or is a new speculative plan generated for each pull request?

**Review by alikhajeh1 — COMMENTED** [review]
Love it! I added my comments. The repo readme and examples/readme pages need to be updated to add links to this too.

File: testdata/terraform-cloud_comment_golden.md (hunk 2 above)
Comment (aliscott) [reply testdata/terraform-cloud_comment_golden.md:62]: Fixed

**Review by alikhajeh1 — APPROVED** [review]

**Pull request merged**